libs/payments/stripe/src/lib/stripe.client.ts

```diff
@@ -28,6 +28,20 @@ export class StripeClient {
     return this.stripe.customers.retrieve(customerId);
   }
 
+  /**
+   * Updates customer object with the values of the parameters passed
+   *
+   * @param customerId The Stripe customer ID of the customer to update
+   * @param params Values to be updated in customer object
+   * @returns The updated customer object or throws an error if paramaters are invalid
+   */
+  async updateCustomer(
+    customerId: string,
+    params: Stripe.CustomerUpdateParams | undefined
```

Comment:
I think `undefined` should be eliminated from the signature here. AFAIK you should not be able to update a customer without passing data to update the customer with.

Reply:
Got it. Thank you! Updated.